v3/src/components/graph/adornments/adornment-content-info.ts

```diff
@@ -5,6 +5,7 @@ export interface IAdornmentContentInfo {
   modelClass: typeof AdornmentModel
   plots: PlotType[]
   prefix: string
+  subTypeOf?: string
```

Comment:
As discussed elsewhere, for now at least, this information does need to be provided in the adornment content info so it's available before an instance of each adornment's model is created. I changed the property name to `parentType`, though, as you suggested. I also added that `isUnivariateMeasure()` view method to the models since it's useful elsewhere.